crates/bevy_app/src/schedule_runner.rs

```diff
@@ -102,10 +102,8 @@ impl Plugin for ScheduleRunnerPlugin {
 
                     #[cfg(not(target_arch = "wasm32"))]
                     {
-                        while let Ok(delay) = tick(&mut app, wait) {
-                            if let Some(delay) = delay {
-                                std::thread::sleep(delay);
-                            }
+                        while let Ok(Some(delay)) = tick(&mut app, wait) {
```

Comment:
This changes the semantics to be wrong, specifically when the runner settings are to run as often as possible

I'm shocked that clippy is suggesting this change to be honest.

Reply:
Should I change it to just ignore this lint then?

Reply:
Yeah, I would just revert and #allow.

See rust-lang/rust-clippy#7575